packages/db/src/query/sensor.ts

```diff
@@ -597,6 +597,22 @@ export async function assignSensorToUser(clientId: string, userId: string | null
             throw new Error("sensor/not-found");
         }
 
+        if (userId) {
+            const userSensorOfSameType = await trx
```

Comment:
Kann man analog und digital gleichzeitig haben oder warum of Same Type?

Reply:
Type ist hier eher `Strom` oder `Gas`. Wir haben Gas ja schon in der DB abgebildet, auch wenn wir das (noch) nicht brauchen